utilities/prm/navier_stokes/ALE_test.prm

```diff
@@ -0,0 +1,158 @@
+
+subsection ALE Navier Stokes Interface
+  set AMG d - use inverse operator     = true
+  set AMG u - use inverse operator     = true
+  set AMG v - use inverse operator     = true
+  set Block of differential components = 1,1,1,0
+  set Blocking of the finite element   = d,d,v,v,u,u,p
+  set Finite element space             = FESystem[FE_Q(1)^d-FE_Q(1)^d-FE_Q(2)^d-FE_Q(1)]
```

Comment:
These are not compatible spaces. I'd keep the same fevalues for displacements and velocities.